test/powershell/Modules/Microsoft.PowerShell.Utility/FormatHex.Tests.ps1

```diff
@@ -5,219 +5,460 @@
-        ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        $actualResult = $result.ToString() -split "`r`n"
-        $actualResult.Count | should be 2
-        $actualResult[0].ToString() | Should be "00000000   61 61 61 61 61 61 61 61 61 61 61 61 61 61 61 61  aaaaaaaaaaaaaaaa"
-        $actualResult[1].ToString() | Should be "00000010   61 61 61 61 61 61 61 61 61 61 61 61 61 61        aaaaaaaaaaaaaa  "
-    }
+    Context "Cmdlet Functionality" {
+
+        It "Path is default Parameter Set 'fhx `$inputFile1'" {
 
-    # This test is to validate that files do not have buffer underrun problems
-    It "ValidateEachBufferHasCorrectContentForFiles" {
-        $result = Format-Hex -path $InputFile2
-        $result | Should Not BeNullOrEmpty
-        $result.Count | should be 2
-        $result[0].ToString() | Should be "00000000   54 68 69 73 20 69 73 20 61 20 62 69 74 20 6D 6F  This is a bit mo"
-        if ( $IsCoreCLR ) {
-            $result[1].ToString() | Should be "00000010   72 65 20 74 65 78 74                             re text         "
+            $result =  Format-Hex $inputFile1
+
+            $result | Should Not BeNullOrEmpty
+            ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
+            $actualResult = $result.ToString()
+            ($actualResult -match $inputText1) | Should Be $true
         }
-    }
 
-    # This test ensures that if we stream bytes from a file, the output is correct
-    It "ValidateStreamOfBytesFromFileHasProperOutput" {
-        $result = Get-Content $InputFile1 -Encoding Byte | Format-Hex
-        $result | Should Not BeNullOrEmpty
-        ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        if ( $IsCoreCLR ) {
-            $result.ToString() | Should be    "00000000   48 65 6C 6C 6F 20 57 6F 72 6C 64                 Hello World     "
+        It "Validate file input from Pipeline 'Get-ChildItem `$inputFile1 | Format-Hex'" {
+
+            $result = Get-ChildItem $inputFile1 | Format-Hex
+
+            $result | Should Not BeNullOrEmpty
+            ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
+            $actualResult = $result.ToString()
+            ($actualResult -match $inputText1) | Should Be $true
         }
-    }
 
-    # This test is to validate the alias for Format-Hex cmdlet.
-    It "ValidateCmdletAlias" {
+        It "Validate that streamed text does not have buffer underrun problems ''a' * 30 | Format-Hex'" {
+
+            $result = "a" * 30 | Format-Hex
 
-        try
-        {
-            $result = Get-Command fhx -ErrorAction Stop
             $result | Should Not BeNullOrEmpty
-            $result.CommandType.ToString() | Should Be "Alias"
+            ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
+            $actualResult = $result.ToString() -split "`r`n"
+            $actualResult.Count | should be 2
```

Comment:
I am not sure if you need this check here since you are already checking both results below, e.g., $actualResult[0], $actualResult[1]